index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather Report</title>
+    <link href="../styles/index.css" rel="stylesheet" />
+</head>
+<body>
+    <header>
+        <p>Weather Report for the city of <span id=cityName>Seattle</span></p>
+    </header>
+    <main>
+        <section id=temperature class="small_box tempBox">
+            <h2 id=tempTitle>TEMPERATURE</h2>
```

Comment:
You skipped directly to a `<h2>` without having an `<h1>`.  Your heading tags should be able to be used to create a table of contents so it's important not to skip levels.